src/app/request-handlers/guards/end-date-guard.ts

```diff
@@ -7,6 +7,7 @@ export class EndDateGuard extends RequestHandlerGuard {
     }
 
     public async check(onReject: (message: string) => Promise<void>): Promise<void> {
-        if (compareDesc(this.submittedDate, this.endDate) == -1) onReject('Request was submitted after the end date');
+        // TODO: Update reject message
+        if (compareDesc(this.submittedDate, this.endDate) == -1) onReject('Request was submitted after the until date');
```

Comment:
Since we no longer mention "Date" in the column name, can the message just be "Request is submitted too late"?